components/dashboard/src/app/LoggedInApp.tsx

```diff
@@ -0,0 +1,39 @@
+/**
+ * Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { Team, User } from "@gitpod/gitpod-protocol";
+import { FunctionComponent, Suspense, lazy } from "react";
+import { AppRoutes } from "./AppRoutes";
+import { Blocked } from "./Blocked";
+import { AppLoading } from "./AppLoading";
+
+const OAuthClientApproval = lazy(() => import(/* webpackPrefetch: true */ "../OauthClientApproval"));
+
+type LoggedInAppProps = {
+    user: User;
+    teams?: Team[];
+};
+export const LoggedInApp: FunctionComponent<LoggedInAppProps> = ({ user, teams }) => {
```

Comment:
I think this component can probably go away once the 2 TODOs in here are "routerized" - then all it's doing is wrapping the AppRoutes in a Suspense to handle a loading state. At that point I think it'd make sense to just do the Suspense wrapping right in the top level `<App/>` component. Could also shift these 2 TODOs into AppRoutes and remove this layer now probably.